### PR TITLE
Support same-ledger transfers

### PR DESCRIPTION
**Pull request by emschwartz**
Note that this is a breaking change because the second commit changes the return value
Addresses https://github.com/interledger/five-bells-sender/issues/7

### CODE REVIEW

File: src/sender.js [hunk 1]
```diff
@@ -20,28 +20,42 @@ function Sender (params) {
...
-  this.subpayments = await this.pathfinder.findPath(
-    this.source_ledger,
-    this.destination_ledger,
-    this.destination_amount)
+  if (this.source_ledger === this.destination_ledger) {
+    // Same ledger transfer
+    // TODO add an execution_condition here whenever we switch to having
+    // the recipient creat the execution_condition_fulfillment
+    // TODO ledger fees?
+    this.transfers.push({
```
Comment (sentientwaffle) [inline src/sender.js]: I think using this variable for intra-ledger transfers is confusing -- its used later to build a list of all transfers.

**Comment by sentientwaffle** [comment]
The API change will require corresponding changes in -visualization and -demo.

File: src/sender.js (hunk 1 above)
Comment (emschwartz) [inline src/sender.js]: In the case where the source and destination are on the same ledger there will only be one transfer in that list

**Comment by emschwartz** [comment]
It looks like neither the [visualization](https://github.com/interledger/five-bells-visualization/blob/master/controllers/payments.js#L17) nor the [demo](https://github.com/interledger/five-bells-demo/blob/5b6230c5fedb90e46b4d51721e062bf8c6c0f25c/scripts/send.js#L14) use the return value from the `send` function, which is the only part of the API that's changing

**Comment by emschwartz** [comment]
Closing this PR for now because same-ledger transfers should be handled separately from multi-hop payments. That should be added after some refactoring

**Pull request closed without merging**